client/my-sites/email/gsuite-purchase-features/index.jsx

```diff
@@ -27,10 +27,15 @@ class GSuitePurchaseFeatures extends React.Component {
 
 	getStorageText() {
 		const { productSlug, translate } = this.props;
-		if ( 'gapps' === productSlug ) {
-			return translate( 'Get 30GB of storage for all your files synced across devices.' );
-		} else if ( 'gapps_unlimited' === productSlug ) {
-			return translate( 'Get unlimited storage for all your files synced across devices.' );
+		switch ( productSlug ) {
```

Comment:
Why switch to a case statements, will it ever not equal 'gapps' or 'gapps_unlimited'

Reply:
The `default` case is used is used in the A/B test to show "30GB or unlimited" to imply the amount of storage is dependent on the sku chosen. The linter was giving me a really hard time about using `if ... else if ... else` with each having a return, so I just switched to a case statement.